src/lib/stores/sdk.ts

```diff
@@ -68,6 +69,7 @@ export const sdk = {
         health: new Health(clientConsole),
         locale: new Locale(clientConsole),
         projects: new Projects(clientConsole),
+        auth: new Auth(clientConsole),
```

Comment:
@TorstenDittmann is there a way to combine this into the projects class ? Because that is where this method is supposed to be

Reply:
once the spec is generated and a sdk is released it will be automatically part of that 👍🏻